corehq/apps/app_manager/models.py

```diff
@@ -1088,8 +1093,8 @@ def generate_paths():
                     actions = action
                 else:
                     actions = [action]
-                for action in actions:
-                    for path in FormAction.get_action_paths(action):
+                for act in actions:
+                    for path in FormAction.get_action_paths(act):
```

Comment:
(This change isn't related to usercase subcases. `action` is also the name of the outer loop variable. I assume it's not meant to be reassigned inside the loop, so I named the inner loop variable `act` instead.)